pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.8.2"
+    version: "2.8.1"
```

Comment:
We don't wanna downgrade any of the packages so this file should probably stay unchanged

Reply:
:+1: fixed